packages/remix-scripts/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@remix-run/scripts",
+  "version": "1.5.1",
+  "description": "CLI scripts for Remix",
+  "homepage": "https://remix.run",
+  "bugs": {
+    "url": "https://github.com/remix-run/remix/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/remix-run/remix",
+    "directory": "packages/remix-scripts"
+  },
+  "license": "MIT",
+  "bin": {
+    "remix": "cli.js"
+  },
+  "dependencies": {
+    "@esbuild-plugins/node-modules-polyfill": "^0.1.4",
+    "@npmcli/package-json": "^2.0.0",
+    "@remix-run/dev": "1.5.1",
```

Comment:
How would you imagine the new `convert-to-javascript` migration to be run from within `@remix-run/dev` if people choose to do so with the CLI prompt?

This would cause a circular dependency if we keep it this way.

Reply:
It would no longer be available under `@remix-run/dev`. The idea is to get all of this out of that package, and the script would change as a result (`remix-scripts migrate`). The `convert-to-javascript` stuff is outside the scope of `remix migrate`. Right now the only command under `migrate` is `https://github.com/remix-run/remix/blob/main/packages/remix-dev/cli/migrate/migrations/index.ts`. https://github.com/remix-run/remix/blob/main/packages/remix-dev/cli/migrate/migrations/index.ts

Reply:
At least for backwards compatibility, we should keep `remix migrate` imo.

This can then run `remix-scripts migrate` under the hood & even first show a warning message, but we can't just remove it I think.

Reply:
The idea behing the new `convert-to-javascript` migration/script was also to move it to this new `@remix-run/scripts` package btw.

That's why @kentcdodds was suggesting `scripts` as package name, as it covers both `migrations` and other scripts.

Having it in `@remix-run/dev` would mean we would copy/paste a lot of the migration code we now have in this new package too.

Reply:
Sorry, what I meant is that we aren't touching the existing JS conversion script that's already in the `dev` package *in this PR*. Step one, I believe, is just to move the `migrate` script out of the `dev` package. This lets us shed some of the weight of dependencies of a core module and keeping utility scripts that won't be used very often in a separate package.

I see the `convert-to-javascript` script you started as additive rather than replacing the current behavior in `dev`. It should go in this new package, but that doesn't mean we need to change the current behavior in `dev` at the moment.  Ultimately I think @kentcdodds will be the final word on this, but that's my take.

In the long run I think it might make sense to move other scripts to this package. It's a little tricky right now since the build/dev scripts are sort of intertwined with our compiler logic, so I think that'll need to be cleaned up first. But alternatively, those scripts are necessary for remix development, whereas other scripts are less used utilities. I don't think the scope for each package is clearly defined yet, hence why I'm just starting with the one script.

As for back-compat, I don't think that's a huge concern for the migrate script. Again, Kent will say for sure. He's out next week so we likely won't merge this until he's back anyway.

Reply:
I'm fine with whatever is decided, just wanted to state my concerns/remarks & make sure we take everything into account 👍